soc/espressif/esp32/CMakeLists.txt

```diff
@@ -24,13 +26,24 @@ zephyr_library_sources_ifdef(CONFIG_POWEROFF poweroff.c)
 # get flash size to use in esptool as string
 math(EXPR esptoolpy_flashsize "${CONFIG_FLASH_SIZE} / 0x100000")
 
-# This includes CONFIG_MCUBOOT and CONFIG_ESP_SIMPLE_BOOT
-if(NOT CONFIG_BOOTLOADER_MCUBOOT)
+# Get UART baudrate from DT
+dt_chosen(dts_shell_uart PROPERTY "zephyr,shell-uart")
+if(${dts_shell_uart})
+  dt_prop(monitor_baud PATH ${dts_shell_uart} PROPERTY "current-speed")
+endif()
+
+board_runner_args(esp32 "--esp-monitor-baud=${monitor_baud}")
+
+message("-- Espressif HAL path: ${ESP_IDF_PATH}")
```

Comment:
seems like a debug leftover

Reply:
In fact, I put it there for a reason. Same line is in the S3 port.